src/service/url-expander/expander.js

```diff
@@ -266,19 +268,20 @@ export class Expander {
       binding = bindingInfo.async || bindingInfo.sync;
     }
     return this.sync_ ?
```

Comment:
@lannka so I thought we wanted to do something like ```
const result = this.sync_ ?
      this.evaluateBindingSync_(binding, name, noEncode, opt_args) :
      this.evaluateBindingAsync_(binding, name, noEncode, opt_args);

    return encode ? encodeURIComponent(result) : result
```

but since this can be a promise (or not) it winds up being something like
```
if (this.sync_) {
      const result = this.evaluateBindingSync_(binding, name, noEncode, opt_args);
      return encode ? encode(result) : result;
    } else {
      return this.evaluateBindingAsync_(binding, name, noEncode, opt_args)
        .then(result => encode ? encode(result) : result;)
    }
```
and I am less sure that it is better than what we have.

Reply:
Done.